### PR TITLE
kadane's algorithm

### PR DESCRIPTION
**Pull request by zenithexpo**
Addressing issue #178 
Language: Python

### CODE REVIEW

File: Code/Python/kadane_algorithm.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#kadane's algorithm 
+#Given any array of integers, the algorithm to find the maximum subarray sum is called Kadane’s Algorithm.
+#Problem statement: Given Array Of N integers a1, a2, a3, ….. aN, find the maximum sum of sub-array of a given array.
+
+#The basic idea of kadane's algorithm is to look for positive contigious sub-array, here current_sum keeps track of it.
+#In each loop we compare the current_sum with best_sum (i.e Maximum Sum) and update if current becomes larger than best_sum.
+#Time complexity: O(n) and Space Complexity: O(1)
```
Comment (plazzy99) [inline Code/Python/kadane_algorithm.py:7]: shift this complexity to the bottom of the file

**Review by plazzy99 — CHANGES_REQUESTED** [review]
Update the readme for the same

File: Code/Python/kadane_algorithm.py (hunk 1 above)
Comment (zenithexpo) [reply Code/Python/kadane_algorithm.py:7]: okay, updating it

**Review by plazzy99 — CHANGES_REQUESTED** [review]
Readme file needs to be updated for the same

**Comment by zenithexpo** [comment]
> Readme file needs to be updated for the same

okay, updating .

**Comment by zenithexpo** [comment]
Should I create a new pull request to update the readme as this branch is  forked of old version of main branch.

**Pull request closed without merging**